src/streams/SubscriptionsScreen.js

```diff
@@ -28,14 +33,47 @@ const styles = createStyleSheet({
     flex: 1,
     flexDirection: 'column',
   },
+  rightItem: {
+    marginLeft: 'auto',
+  },
+  rightIcon: {
+    marginLeft: 'auto',
+    color: '#666699',
+  },
+  allStreamsButton: {
+    paddingRight: 12,
+  },
+  streamsText: {
+    textTransform: 'uppercase',
+    color: '#666699',
```

Comment:
Interesting, was this color specified in the design prototype? I don't think I'd seen specific colors mentioned, except in a couple of places related to stream colors.

When I switch to a dark theme, it looks like this color doesn't get good contrast. Presumably as the new design gets to a later stage, Vlad will work out a pair of colors to use here for the light and dark themes.

For now, let's do the same thing here as `NestedNavRow` does and use the theme's `color` property. (In `NestedNavRow`, this appears directly for the icons, and is handled for the text down in `ZulipText`.) That's a color chosen to have good contrast against the respective theme's background color.

Reply:
Ah, ya I probably should have asked about that. I used an eyedropper tool to pull the color from the prototype. Sounds good.